src/wwwroot/styles/modules/_copyToClipboard.scss

```diff
@@ -0,0 +1,7 @@
+.copyContainer .inputContainer{
```

Comment:
We are currently using BEM methodology (Block, Element, Modifier), lower camel case aren't allowed in our sass files.